.gitignore

```diff
@@ -40,3 +40,6 @@ main.js.map
 
 # Generated translation files
 locales
+
+# Generated by the backend
+app/api/ClientApi.js
```

Comment:
I think something like `CardanoWalletApi` (how will it work in the future with other currencies etc.?) would be more meaningful in the context of the frontend app. (E.g: we already have a `CardanoClientApi` in the same folder … which could be confusing).